medical_lab/models/medical_lab_test_criterion.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Copyright 2012-2013 Federico Manuel Echeverri Choux
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models
+
+
+class MedicalLabTestCriterion(models.Model):
+
+    _name = 'medical.lab.test.criterion'
+    _description = 'Medical Lab Test Criteria'
+    _order = 'sequence'
+
+    name = fields.Char(
+        string='Test Name',
```

Comment:
Do we need to explicitly define **Test Name** instead of just letting Odoo use **Name**? Not sure. I guess since other fields have 'Test' in their string attributes having Test Name makes sense.